### ISSUE TITLE
Undefined subroutine &Test::WWW::Mechanize::assert_arrayref

### ISSUE THREAD

**Issue by eserte** [petdance/test-www-mechanize#65]
On some of my smoker systems the test suite fails:
```
Undefined subroutine &Test::WWW::Mechanize::assert_arrayref called at /home/cpansand/.cpan/build/2018120521/Test-WWW-Mechanize-1.52-g5g_bS/blib/lib/Test/WWW/Mechanize.pm line 1703.
# Looks like you planned 23 tests but ran 4.
# Looks like your test exited with 255 just after 4.
t/id_exists.t .............. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 19/23 subtests 
```
It may be related to older versions of Carp::Assert::More being installed on these systems, but statistical analysis is not 100% clear about it.

**Comment by petdance** [comment]
That's undoubtedly what it is.  I thought I had that in the prereqs.  Sorry about that.  Thanks for catching.

**Comment by petdance** [comment]
Aha.  It needs a newer version of Carp::Assert::More.

Fixed.  Thank you.  9f6c5ce

**Comment by castaway** [comment]
The patch is lovely.. Ive just stumbled on this on the latest CPAN version of Test::WWW::Mechanize, just checked and it still says Carp::Assert::More => 0.. any chance of releasing this?

**Comment by petdance** [comment]
I've just released 1.54 to CPAN.  Thanks for the nudge.
